includes/fbproduct.php

```diff
@@ -662,6 +662,31 @@ public function prepare_product( $retailer_id = null, $type_to_prepare_for = sel
 			$brand = wp_strip_all_tags( WC_Facebookcommerce_Utils::get_store_name() );
 		}
 
+		$custom_fields = array(
+			'description' => '',
+			'price' => '',
+			'image_url' => ''
+		);
+
+		// check if fb specific description exists
+		$fb_description = get_post_meta($this->id, self::FB_PRODUCT_DESCRIPTION, true);
+		if (!empty($fb_description)){
```

Comment:
Format, the same on the rest of the code please update for consistent formatting:

```suggestion
		if ( !empty( $fb_description ) ) {
```